semantic_conventions/trace/database.yaml

```diff
@@ -382,6 +382,49 @@ groups:
           The collection being accessed within the database stated in `db.name`.
         examples: [ 'customers', 'products' ]
 
+  - id: db.elasticsearch
+    prefix: db.elasticsearch
+    type: span
+    extends: db
+    brief: >
+      Call-level attributes for Elasticsearch
+    attributes:
+      - id: doc_id
+        type: string
+        requirement_level:
+          conditionally_required: when the request targets a specific document by id
+        tag: call-level-tech-specific-elasticsearch
+        brief: >
+          The document that the request targets.
+        examples: [ '123', '456' ]
+      - id: target
+        type: string
+        requirement_level:
+          conditionally_required: when a specific index or data stream is targeted by the request
+        tag: call-level-tech-specific
+        brief: >
+          The name of the data stream or index that is targeted.
+        examples: [ 'users' ]
+      - ref: url.query
```

Comment:
it's a bit confusing to me to see it as `url.query` even if it's how it's passed in case of elastic.

why not put it into `db.statement`, especially since it's already used to record query when it's passed in the body?

Would it make sense to capture `url.full` to comply with HTTP spec and putting DB semantics into `db.*` attributes?